### ISSUE TITLE
Unable to use pretrained models

### ISSUE THREAD

**Issue by miguel-dgist** [Raykoooo/IAST#3]
Hi, I've been unsuccessfully trying to reproduce your results. Neither train nor test seem to be working on my machine, all the results are almost zero. Here is what I got when I tried testing:

```
$ python eval.py --config_file config/gtav2cityscapes_t4/run_task/sl_3.yaml --resume_from pretrained_models/gta5_t4_best.pth 
Resume from: pretrained_models/gta5_t4_best.pth
Eval Size: [[1024, 512], [1280, 640], [1536, 768], [1800, 900], [2048, 1024]]
Use Flip: False
100%|█████████████████████████████████████████| 250/250 [11:54<00:00,  2.86s/it]
val_miou: 0.0049, 0: 0.0000, 1: 0.0001, 2: 0.0117, 3: 0.0266, 4: 0.0358, 5: 0.0062, 6: 0.0000, 7: 0.0000, 8: 0.0001, 9: 0.0054, 10: 0.0000, 11: 0.0020, 12: 0.0008, 13: 0.0008, 14: 0.0000, 15: 0.0003, 16: 0.0000, 17: 0.0029, 18: 0.0000
```

I will appreciate your suggestions!

**Comment by Raykoooo** [comment]
Hi, This seems to be an error in the label you read. If you have not modified any Dataloader related code, please check whether your cityscapes image and label are correct.

**Comment by miguel-dgist** [comment]
Thank you for your answer. Actually the only issue I had was that the name of cityscapes labels has a "Train" word added to the original ones I downloaded from cityscapes website.

The file "IAST/data/cityscapes_train.json" has:
"mask_name": "gtFine/train/aachen/aachen_000000_000019_gtFine_labelTrainIds.png"

while the data I got came with the name:
"gtFine/train/aachen/aachen_000000_000019_gtFine_labelIds.png"

Is there any preprocessing step that actually modifies these labels?

**Comment by Raykoooo** [comment]
> The file "IAST/data/cityscapes_train.json" has:
> "mask_name": "gtFine/train/aachen/aachen_000000_000019_gtFine_labelTrainIds.png"
> 
> while the data I got came with the name:
> "gtFine/train/aachen/aachen_000000_000019_gtFine_labelIds.png"
> 
> Is there any preprocessing step that actually modifies these labels?

Please refer to https://github.com/mcordts/cityscapesScripts.

**Comment by miguel-dgist** [comment]
It's working now. Thank you!

**Comment by akshaykulkarni07** [comment]
Hi, I am facing the same issue. I generated the `labelTrainIds` images using [createTrainIdLabelImgs.py](https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/preparation/createTrainIdLabelImgs.py). However, it still gives the same issue (mIoU ~0.009). Could you please let me know how exactly you preprocess the data? Thanks.

**Comment by miguel-dgist** [comment]
Hi @akshaykvnit, I did not perform any additional preprocessing steps but I had only check the Tesla T4 pretrained model.

**Comment by adricarda** [comment]
Hi, thank you for releasing code and checkpoints. However, I am not able to reproduce the results in the repository table (51.88 and 51.2) for GTA. Are these results obtained with multi-scaling? if not what resolution do you use? Other works typically use 1024x512 at test time, but this does not seem to work.

Thank you in advance,
